### ISSUE TITLE
Error - Cannot end a section without first starting one

### ISSUE THREAD

**Issue by pxpm** [Laravel-Backpack/basset#44]
I was working on something when I stumbled upon this. 

So what I was trying to do, was to add a JS file in a field 
```php
@push('crud_fields_scripts')
        @basset(base_path('path_to_my_script.js'))
@endpush
```

It happens that I **wrongly typed the path** without noticing, and when I try to reload my application this is the error I got:
![image](https://github.com/Laravel-Backpack/basset/assets/7188159/af59818f-1c67-47be-9b8c-e4344e500c62)

as you can see from the error trace https://flareapp.io/share/NPLV1235 it's not even remotely close to what the real issue was. 

This needs some improvement or it will end up hurting someone. 😢

**Comment by promatik** [comment]
@pxpm I need your help 🙌
How can I reproduce this one?

From what I understood you just added a fake path(?) on a field and it failed?

I tried to go to `text.blade.php` and add;
```php
@push('crud_fields_scripts')
    @bassets(base_path('fakepath.js'))
@endpush
```
But it doesn't break ...

This is strange because basset blade directives shouldn't interfere with others, like the section one.
Please let me know more details 🙌

**Comment by pxpm** [comment]
What I did was very similar to that, but was in color_picker.

![image](https://github.com/Laravel-Backpack/basset/assets/7188159/dd6cb5a4-8118-4d16-935d-de6fe629686e)

It has JS that text field does not have, it may be that issue.

Let me know if you still not able to reproduce it.

Cheers

**Comment by promatik** [comment]
Fixed by #52 🙌
